src/scripts/components/pages/authors.js

```diff
@@ -1,34 +1,40 @@
 import clearDom from '../../helpers/clearDom';
 import renderToDOM from '../../helpers/renderToDom';
 
+const emptyAuthors = () => {
+  const domString = '<h1>No Authors</h1>';
+  renderToDOM('#store', domString);
+};
+
 const showAuthors = (array) => {
   clearDom();
 
+  const parsedAuthorArray = Object.values(array);
+
   const btnString = '<button class="btn btn-success btn-lg mb-4" id="add-author-btn">Add An Author</button>';
 
   renderToDOM('#add-button', btnString);
 
-  let domString = '';
-  array.forEach((item) => {
-    domString += `
+  if (array.length) {
+    let domString = '';
+    parsedAuthorArray.forEach((item) => {
+      domString += `
     <div class="card" style="width: 18rem;">
       <div class="card-body">
         <h5 class="card-title">${item.first_name} ${item.last_name}</h5>
-        <h6 class="card-subtitle mb-2 text-muted">${item.email}</h6>
+        <p class="card-text bold">${item.favorite ? `<span class="badge badge-info sale-badge"><i aria-hidden="true"></i>Fav</span> $${item.email}` : `$${item.email}`}</p>
```

Comment:
```suggestion
        <p class="card-text bold">${item.favorite ? `<span class="badge badge-info sale-badge"><i aria-hidden="true"></i>Fav</span> ${item.email}` : `${item.email}`}</p>
```

We don't need these extra dollar signs